apps/connect/src/env/usdc-bridge.ts

```diff
@@ -18,23 +19,24 @@ export const ENV: Env = {
-  } as WormholeConnectConfig,
+  wormholeConnectConfig: mergeDeep<WormholeConnectConfig>(
+    wormholeConnectConfigCommon,
+    {
+      ui: {
+        pageHeader: { text: "USDC Transfer", align: "center" },
```

Comment:
if you don't specify a `title` here, Connect will now just say "Wormhole Connect" at the top. I suggest maybe setting it to `title: "USDC Bridge"`

<img width="533" alt="image" src="https://github.com/user-attachments/assets/af0e7f7f-1eac-467a-b6b2-8bc8fbb2c432">

Reply:
that one is deep merging, it will default to the "common" one `Bridge assets`